src/OrchardCoreContrib.PoExtractor/Program.cs

```diff
@@ -102,7 +121,7 @@ public static void Main(string[] args)
                     projectProcessor.Process(projectPath, projectBasePath, localizableStrings);
                 }
 
-                if (isSingleFileOutput)
+                if (!isSingleFileOutput)
```

Comment:
This is fixed in another PR, right?

Reply:
Yes. I merged from your `main` branch and now it's not shown as a diff in this PR.